sound/soc/sof/ipc4-topology.c

```diff
@@ -491,8 +491,10 @@ static int sof_ipc4_widget_setup_comp_dai(struct snd_sof_widget *swidget)
 		ipc4_copier->data.gtw_cfg.node_id |=
 			SOF_IPC4_NODE_INDEX_INTEL_SSP(ipc4_copier->dai_index);
 		break;
+	case SOF_DAI_INTEL_DMIC:
+		/* nothing to do for DMIC */
+		break;
```

Comment:
Do we need to set node_id for DMIC here? We may have two DMICs, one for 48K and the other for 16K.

Reply:
@bardliao that would be my guess too but since we only have one dmic in tplg, we can't test it and I don't want to add it without testing